source/js/components/post-code-validation.js

```diff
@@ -0,0 +1,58 @@
+import { countriesAndPostCodes } from "./post-codes-list.js";
+
+// List of countries to use for reference
+const listOfCountries = countriesAndPostCodes;
+
+// The container for the post-code input field
+const postCodeInputContainer = document.querySelector(
+  ".post-code-input-container"
+);
+
+// The container that houses both the post code and city inputs
+const formAndCityContainer = document.querySelector(".form__group--city-post");
+
+// The select object that allows users to select their country
+const countrySelector = document.getElementById("id_country");
+
+// Calling the function once to check in case the user refreshes the page,
+// we want the input to remain one column if needed
+if (countrySelector) {
```

Comment:
The if statements on line 19 and 24 are verifying that the country select input exists on the page, if not, no need to run the function or add event handler yet

Reply:
If so, you'll want to move Lines 11 and 14 to inside the `if` block, because there's no reason to query for them if countrySelector is nothing.

Reply:
Awesome catch Pomax!